server/pages/browsehappy.jade

```diff
@@ -0,0 +1,68 @@
+		meta(name='format-detection', content='telephone=no')
+		meta(name='mobile-web-app-capable', content='yes')
+		meta(name='apple-mobile-web-app-capable', content='yes')
+		meta(name='referrer', content='origin')
+
+		link(rel='shortcut icon', type='image/vnd.microsoft.icon', href=faviconURL, sizes='16x16 32x32')
+		link(rel='shortcut icon', type='image/x-icon', href=faviconURL, sizes='16x16 32x32')
+		link(rel='icon', type='image/x-icon', href=faviconURL, sizes='16x16 32x32')
+		link(rel='icon', type='image/png', href='//s1.wp.com/i/favicons/favicon-64x64.png', sizes='64x64')
+		link(rel='icon', type='image/png', href='//s1.wp.com/i/favicons/favicon-96x96.png', sizes='96x96')
+		link(rel='icon', type='image/png', href='//s1.wp.com/i/favicons/android-chrome-192x192.png', sizes='192x192')
+		link(rel='apple-touch-icon', sizes='57x57', href='//s1.wp.com/i/favicons/apple-touch-icon-57x57.png')
+		link(rel='apple-touch-icon', sizes='60x60', href='//s1.wp.com/i/favicons/apple-touch-icon-60x60.png')
+		link(rel='apple-touch-icon', sizes='72x72', href='//s1.wp.com/i/favicons/apple-touch-icon-72x72.png')
+		link(rel='apple-touch-icon', sizes='76x76', href='//s1.wp.com/i/favicons/apple-touch-icon-76x76.png')
+		link(rel='apple-touch-icon', sizes='114x114', href='//s1.wp.com/i/favicons/apple-touch-icon-114x114.png')
+		link(rel='apple-touch-icon', sizes='120x120', href='//s1.wp.com/i/favicons/apple-touch-icon-120x120.png')
+		link(rel='apple-touch-icon', sizes='144x144', href='//s1.wp.com/i/favicons/apple-touch-icon-144x144.png')
+		link(rel='apple-touch-icon', sizes='152x152', href='//s1.wp.com/i/favicons/apple-touch-icon-152x152.png')
+		link(rel='apple-touch-icon', sizes='180x180', href='//s1.wp.com/i/favicons/apple-touch-icon-180x180.png')
+		link(rel='profile', href='http://gmpg.org/xfn/11')
+		link(rel='manifest', href='/calypso/manifest.json')
+		link(rel='stylesheet', href='//s1.wp.com/i/fonts/merriweather/merriweather.css?v=20160210')
+		link(rel='stylesheet', href='//s1.wp.com/i/noticons/noticons.css?v=20150727')
+		link(rel='stylesheet', href='//s1.wp.com/wp-includes/css/dashicons.css?v=20150727')
+		if isRTL
+			link(rel='stylesheet', href=urls['style-rtl.css'])
+		else
+			if 'development' === env || isDebug
+				link(rel='stylesheet', href=urls['style-debug.css'])
+			else
+				link(rel='stylesheet', href=urls['style.css'])
+
+	body(class=isRTL ? 'rtl' : '')
+		#wpcom.wpcom-site
+			.layout.has-no-sidebar
+				#content.layout__content
+					#primary.layout__primary
+						main.browsehappy__main.main(role="main")
+							.empty-content
+								img.empty-content__illustration(src="/calypso/images/drake/drake-browser.svg")
+								h2.empty-content__title Unsupported Browser
+								h3.empty-content__line
+									| Unfortunately this page cannot be used by your browser. You can either 
+									a(
+										href="https://dashboard.wordpress.com/wp-admin/"
```

Comment:
You're right. :)

> Because any server-side rendering in Calypso must be a logged-out operation

This was before using Jade, when dealing with a pseudo-isomorphic section — because of caching of the server output. I was starting to use the bootstrapped object when I stopped because of the whole iso thing. 😄